.clitest/functions/.test.setup

```diff
@@ -0,0 +1,17 @@
+#!/bin/zsh
+
+# initialize znap like a user would
+source $PWD/znap.zsh
+
+# works with BSD and GNU gmktemp
+T_TEMPDIR="$(mktemp -d -t t_znap.XXXXXXXX)"
+T_TEMPDIR=${T_TEMPDIR:A}
+
+# set znap location
+hash -d znap=$T_TEMPDIR
+
+# save fpath
+fpath_=( "$fpath[@]" )
```

Comment:
The quotes and `[@]` are not necessary, unless you want to preserve empty strings:
```zsh
fpath_=( $fpath )
```

Reply:
LOL, I copied that from your tests, so I figured that was your preferred style. Happy to change it, but do you want me to change it everywhere?

https://github.com/marlonrichert/zsh-snap/blob/9b8edd89c2b2303ffa253c5116a97d9138e0831d/.clitest/znap.prompt.load.md?plain=1#L4